tests/test_collections.py

```diff
@@ -343,50 +343,3 @@ def test_feature_collection_with_dates_serializes_correctly():
         assert fc.schema == schema
         assert fc[0].geometry == feature.geometry
         assert fc[0].attributes == expected_attributes
-
-
-def test_get_tile_that_returend_257():
```

Comment:
These tests were testing `get_tile`, so I'm removing them as well with the hope that we move them somewhere else.